<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowTrackListEditor</class>
 <widget class="QDialog" name="WindowTrackListEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Track database</string>
  </property>
  <widget class="QListWidget" name="lb_Tracks">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>461</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Track">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Track:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lb_TrackConfigs">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>461</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_TrackConfigs">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Available configurations for track:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_SelectedTrackId">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selected track ID:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_SelectedTrackIdInput">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_SelectedConfigId">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Selected configuration ID:</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_SelectedConfigIdInput">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>420</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="gr_AddNewTrack">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>461</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Add new track:</string>
   </property>
   <widget class="QLabel" name="lbl_AddNewTrack_TrackName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Track name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_AddNewTrack_NameInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddNewTrack_Add">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_DeleteTrack">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete selected track</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_DeleteTrackConfig">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete selected track config</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gr_AddNewTrackConfig">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Add new track configuration:</string>
   </property>
   <widget class="QLabel" name="lbl_AddNewTrackConfig_ConfigName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Configuration name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_AddNewTrackConfig_NameInput">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddNewTrackConfig_Add">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_AddNewTrackConfig_Length">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Length (m):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_AddNewTrackConfig_LengthInput">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_Select">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
